### PR TITLE
Bugfix/display map

### PR DESCRIPTION
**Pull request by giovanneguerra**
Display map again in details page

### CODE REVIEW

**Comment by smakhtin** [comment]
@giovanneguerra something wrong with the used library, it's crashes whole app when used on vercel edge functions. I tried to build it just for node server, same problems:

```
ERROR  (node:20414) [DEP0151] DeprecationWarning: No "main" or "exports" field defined in the package.json for /home/vadim/projects/identifier/node_modules/@studiometa/vue-mapbox-gl/ resolving the main entry point "index.js", imported from /home/vadim/projects/identifier/apps/explorer/node_modules.
```

File: apps/explorer/components/SNIMap.vue [hunk 1]
```diff
@@ -1,134 +1,33 @@
...
-      const el = document.createElement('div')
-      el.id = 'marker'
-
-      // create the marker
-      new Marker(el).setLngLat(this.deCeuvel).setPopup(popup).addTo(map)
-    }
-  }
-})
+const accessToken =
+  'pk.eyJ1IjoiZ2lvdmFubmVndWVycmEiLCJhIjoiY2w2bTFkOHd2MGg2MjNicnVyNHR4ZG5mYiJ9.MouD-LX3BD8dH-pewUSHTw'
```
Comment (smakhtin) [inline apps/explorer/components/SNIMap.vue:31]: Please move access token to ENV variable.

File: apps/explorer/package.json [hunk 2]
```diff
@@ -12,11 +12,13 @@
     "preview": "nuxt preview"
   },
   "devDependencies": {
+    "@mapbox/mapbox-gl-geocoder": "^5.0.1",
```
Comment (smakhtin) [inline apps/explorer/package.json:15]: Why we need geocoder dependency?

**Review by smakhtin — CHANGES_REQUESTED** [review]
Small refactoring required before we merge.

File: apps/explorer/package.json (hunk 2 above)
Comment (giovanneguerra) [reply apps/explorer/package.json:15]: https://vue-mapbox-gl.studiometa.dev/guide/installation/

Peer dependency

Comment (smakhtin) [reply apps/explorer/package.json:15]: @giovanneguerra For us it's not needed since we are not using geocoding component.

**Review by smakhtin — CHANGES_REQUESTED** [review]

**Pull request merged**